src/partials/hero.html

```diff
@@ -1 +1,19 @@
-<section>hero</section>
+<section class="hero section">
+  <div class="container">
+    <div class="hero-container">
```

Comment:
add `hero-container` class to the parent div
like this
`<div class="container hero-container">`